ygen/enumgen_test.go

```diff
@@ -169,6 +317,240 @@ func TestFindEnumSet(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "simple enumeration with conflicting names",
+		in: map[string]*yang.Entry{
+			"/container/state/enumeration-leaf": {
+				Name: "enumeration-leaf",
+				Type: &yang.YangType{
+					Name: "enumeration",
+					Enum: &yang.EnumType{},
+				},
+				Node: &yang.Enum{
+					Name: "enumeration-leaf",
+					Parent: &yang.Container{
+						Name: "state",
+						Parent: &yang.Container{
+							Name: "container",
+							Parent: &yang.Module{
+								Name: "base-module",
+							},
+						},
+					},
+				},
+				Parent: &yang.Entry{
+					Name: "state",
+					Parent: &yang.Entry{
+						Name:   "container",
+						Parent: &yang.Entry{Name: "base-module"},
+					},
+				},
+			},
+			"/container/state/enumerationLeaf": {
```

Comment:
Ditto here re: camelcasing being the same.

Reply:
same: I won't do this the next PR will render this test obsolete.